public/docs/getting_started/javascript-tests.md

```diff
@@ -83,6 +83,61 @@ contract('MetaCoin', function(accounts) {
 });
 ```
 
+## Example (using async/await)
+```
+const MetaCoin = artifacts.require("MetaCoin");
+
+contract('2nd MetaCoin test', async (accounts) => {
+
+  it("should put 10000 MetaCoin in the first account", async() => {
+     let instance = await MetaCoin.deployed();
+     let balance = await instance.getBalance.call(accounts[0]);
+     assert.equal(balance.valueOf(), 10000);
+  })
+  
+  it("should call a function that depends on a linked library", async() => {
+    let instance = await MetaCoin.deployed();
+    let meta = instance;
+    let outCoinBalance = await meta.getBalance.call(accounts[0]);
+    let metaCoinBalance = outCoinBalance.toNumber();
```

Comment:
Might be worth a comment here saying that it's a BigNumber, just so `toNumber()` is more clear